content/01.main_content.md

```diff
@@ -1,36 +1,37 @@
+  Furthermore, the differences between journalists was almost completely removed when considering people with a US-affiliated paper cited in the article.
+  This suggests a regional bias, arguing for more regionally focused journalists.
 
-Audits of representation would ideally be done with all individuals self-reporting gender, ethnicity, and other identifications [@doi:10.1371/journal.pcbi.1003903].
-  While this may be possible prospectively, surveying for self-identification is impractical for large groups and often impossible retrospectively [@http://mediashift.org/2014/11/how-to-ethically-and-responsibly-identify-gender-in-large-datasets/].
-  Computationally-derived predictions allow for audits on a scale that would not be possible otherwise.
-  Numerous tools exist to algorithmically infer gender, nationality, and ethnicity information using only the feature most likely to be non-missing in a relevant dataset: an individual's name. [@doi:10.1093/pan/mpw001; @http://genderize.io/]
-  Most of these models are highly scalable, allowing auditors to define the scope of their target group and background population as broadly as needed.
+Ideally, audits would only use self-reported gender, ethnicity, and other identifications [@doi:10.1371/journal.pcbi.1003903].
+  While this is possible prospectively, surveying is impractical for large groups and often impossible retrospectively [@http://mediashift.org/2014/11/how-to-ethically-and-responsibly-identify-gender-in-large-datasets/].
+  In contrast, computationally-derived predictions allow for large-scale audits.
+  Numerous tools exist to algorithmically infer gender, nationality, and ethnicity using the feature likely to be present in most datasets: an individual's name. [@doi:10.1093/pan/mpw001; @http://genderize.io/]
+  These models are highly scalable, allowing auditors to broadly define the scope of their audit.
   
-Prediction models are not a panacea; several factors limit both their accuracy at recapitulating self-reported information and their ability to address the underlying motivating questions of diversity audits.
+Prediction models are not a panacea; several factors limit both their accuracy and utility.
   For instance, gender associations of a given name can vary by culture, potentially biasing gender predictions where additional information is not available [@https://aclanthology.org/U14-1021].
   Also, most gender prediction models are trained on binary gender labels, which occludes assessing the representation of transgender, non-binary, and intersex individuals [@doi:10.1101/2020.10.12.336230].
  
-Proxy predictions of ethnicity via name origin are more difficult still; choosing categories to probabilistically predict on is non-trivial and difficult to define. 
+Proxy predictions of ethnicity via name origin are more difficult still; defining categories needed for computational analyses is non-trivial and difficult to define. 
```

Comment:
I'm not sure this part is communicating the same information anymore. What prompted the change?

Reply:
ah, yes... i think its better before actually. I was rewording it a lot, but ended up with almost the same number of words. I will revert.

Reply:
reverted in https://github.com/greenelab/disparities_opinion/pull/10/commits/aa299fe5b1ac6c57f203ce5386fca40aec0e9510